edx_django_utils/ip/README.rst

```diff
@@ -0,0 +1,17 @@
+Request Utils
```

Comment:
IP Utils?

Reply:
Ah yes, thank you -- that was an artifact from an earlier version of this branch when I was considering having this be a broader "request utils" module.